app/jinja_filters.py

```diff
@@ -193,6 +193,32 @@ def should_wrap_with_fieldset_processor():
     return {"should_wrap_with_fieldset": should_wrap_with_fieldset}
 
 
+@blueprint.app_template_filter()
+def get_width_class_for_number(answer):
+    allowable_widths = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 20]
+
+    min_value = answer.get("min_value", {}).get("value", 0)
+    max_value = answer.get("max_value", {}).get("value")
+    if not max_value:
+        return None
```

Comment:
Should this not default to the length of our system max default: `MAX_NUMBER = 9999999999`

Reply:
I did think about this, but was reluctant as it seemed like a business survey limitation. Having said that, it is a limit elsewhere so should be used here. I've moved `MAX_NUMBER` into `app.settings` so it can be accessed from both places.